CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ### Bugfixes
 
 * Lorem ipsum.
+### Internals
+
+* Implemented inter-process CondVars on Windows (Win32 + UWP). They should be fair and
+  robust.
```

Comment:
Missed my eye due to the vertical "-----" which separated it from the rest. Why is that line there? :)